README.md

```diff
@@ -13,6 +13,7 @@ Current revision: `stdx` 0.118.0-rc, for Rust 1.18, June 8, 2017.
 | Date and time                  | [`chrono = "0.3.1"`]       | [📖][d-chrono]      |
 | Command-line argument parsing  | [`clap = "2.24.2"`]        | [📖][d-clap]        |
 | Error handling                 | [`error-chain = "0.10.0"`] | [📖][d-error-chain] |
+| FNV hash function              | [`fnv = "1.0.3"`]          | [📖][d-fnv]         |
```

Comment:
Since most people won't know what "FNV" is, how about we say "Fast (FN) hashing".